test/IDE/complete_unresolved_members.swift

```diff
@@ -207,7 +207,7 @@ var a = {() in
   OptionSetTaker5([.#^UNRESOLVED_18^#], .Option4, .South, .West)
 }
 var Container = OptionTakerContainer1()
-Container.OptionSetTaker1(.#^UNRESOLVED_19^#
+Container.OptionSetTaker1(.#^UNRESOLVED_19^#)
```

Comment:
What changed here?

Reply:
Ah, I might have just accidentally added that while testing...

Reply:
@nathawes Any idea why having paren after `UNRESOLVED_19` be affecting results of `UNRESOLVED_20`? I tested again - if I add paren there `UNRESOLVED_20` produces results, otherwise it doesn't, shouldn't these be distinct expressions?

Reply:
I have talked to @nathawes about this and we've looked at AST dumps for both tests, the expected behavior is that both of the expressions here are code-completed separately but without paren they are actually made to be a single expression, which is not what test is trying to verify at all, so it's okay to add paren at the end.